example/gluon/lipnet/README.md

```diff
@@ -1,3 +1,22 @@
+<!---
```

Comment:
This was missed from #13647 even though I made the comment on the PR about the same (see https://github.com/apache/incubator-mxnet/pull/13647#discussion_r256253644)